src/temaWeek6LocalStore/main/LocalStore.java

```diff
@@ -0,0 +1,258 @@
+package temaWeek6LocalStore.main;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Scanner;
+
+
+public class LocalStore {
+	private String name;
+	private double price;
+	private String validityDate;
+	private double weight;
+	private String ID;
+	private String storageTemp;
+	private String vitamin;
+	private int quantity;
+	private String saleDate;
+	private int inputQuantity;
+	private Product prod;
+	
+	List<StockItem> stockItems=new ArrayList<>();
+	List<SoldItem> soldItems=new ArrayList<>();
+	
+	
+	
+	Product p=new Product(name,price,validityDate,weight,ID);
+	StockItem stock=new StockItem(new Animal(name,price, validityDate,weight, ID,storageTemp),quantity);
+	StockItem sold=new StockItem(new Animal(name,price, validityDate,weight, ID,storageTemp),quantity);
+	StockItem vegetableItem=new StockItem(new Vegetable(name,price, validityDate,weight, ID,vitamin),quantity);
+	
+	
+	//	menu
+	public void getMenu() {
+		boolean running = true;
+		
+		System.out.println("Welcome to LocalStore!");
+		while (running) {
+			System.out.println("\n Enter your choice: " +//options in the menu
+					"\n 1.Create product and add it to stock" +
+					"\n 2.Sell products" +
+					"\n 3.Display daily sales report" +
+					"\n 4.Exit");
+			
+			
+			Scanner scan = new Scanner(System.in);
+			String option = scan.next();
+			LocalStore lc = new LocalStore();
```

Comment:
you are creating a new object and passing your stockItems as parameters to those methods... if you would just call the other methods on this object then you would already have access to the stockItems attribute